package.json

```diff
@@ -154,13 +158,16 @@
     "pull-stream": "^3.6.8",
     "pull-stream-to-stream": "^1.3.4",
     "pull-zip": "^2.0.1",
+    "pump": "^3.0.0",
     "read-pkg-up": "^4.0.0",
     "readable-stream": "2.3.6",
     "receptacle": "^1.3.2",
     "stream-to-pull-stream": "^1.7.2",
     "tar-stream": "^1.6.1",
     "temp": "~0.8.3",
     "update-notifier": "^2.5.0",
+    "varint": "^5.0.0",
+    "wrtc": "~0.2.1",
```

Comment:
This is deliberately not installed (as it is not needed)

Reply:
Perhaps it should be moved to peerDependencies?